src/App.js

```diff
@@ -1,27 +1,8 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return (<Calculator />);
 }
```

Comment:
- [ ] [REQUIRED] Great implementation @sameh080081 , however, the project requirement states that this component should also be class-based. Kindly implement this to get the project approved.

Reply:
Hello @Whoistolu, class based component causes eslint error